spp_import_match/models/import_match.py

```diff
@@ -66,10 +66,11 @@ def _match_find(self, model, converted_row, imported_row):
                         domain.append((field_value, "=", row_value))
             if not combination_valid:
                 continue
-            _logger.info("DOMAIN: %s" % domain)
             match = model.search(domain)
             if len(match) == 1:
                 return match
+            elif len(match) > 1:
+                raise ValidationError(_(f"Multiple matches found for '{match[0].name}'!"))
```

Comment:
@emjay0921 `_(f"")` is not a valid translation string. Use .format() or percentage format instead.